Documentation/Usage.md

```diff
@@ -393,6 +393,31 @@ import Foundation
 #endif
 ```
 
+## Using Mirrored packages
+
+To add mirror for a given package:
```

Comment:
I think it'll good to add some description of the feature before diving into an example. It'll also be nice to briefly mention the config file.

Reply:
Sounds good. Let me draft a brief description.

Reply:
done.